### DISCUSSION TITLE
Add support for sync_physics to work with PathFollow2D node

### DISCUSSION THREAD

**Discussion by valkyrienyanko** [godotengine/godot-proposals#5317] [2D]
I have created a platform that moves between 2 points and another platform that moves in a circle. Both have `sync_physics` enabled so the player stays with the platform. Having the player stay with a platform that is a child of a PathFollow2D node would be nice. Right now I am thinking of making my own temporary implementation of the PathFollow2D node just so I can have a platform that moves in a complex path with `sync_physics` enabled.

**Comment by Calinou** [comment]
See https://github.com/godotengine/godot/issues/60237.
